<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDial</class>
 <widget class="QDialog" name="DialogDial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>161</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authentification Box</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RImages.qrc">
    <normaloff>:/MesImages/MesImages/Screenshot_20210215-172553.png</normaloff>:/MesImages/MesImages/Screenshot_20210215-172553.png</iconset>
  </property>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>77</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QDial" name="dial">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDial" name="dial2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDial" name="dial3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>61</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumber2">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumber3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Bold&quot;;</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="RImages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogDial</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
